### PR TITLE
Added server sockets

### PR DESCRIPTION
**Pull request by mikolertesx**

### CODE REVIEW

File: packages/frontend/.eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-	"extends": ["next/babel", "next/core-web-vitals"]
+	"extends": ["next"]
```
Comment (mikolertesx) [inline packages/frontend/.eslintrc.json:2]: This is supposed to be working now; https://stackoverflow.com/questions/71662525/failed-to-load-config-next-babel-to-extend-from-eslintrc-json

File: packages/frontend/next.config.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
 
-module.exports = nextConfig
+// Allows us to use sockets
+const nextConfig = {
+  transpilePackages: ["bufferutil", "utf-8-validate"],
+};
```
Comment (mikolertesx) [inline packages/frontend/next.config.js:6]: I don't really understand why we have to do this, but it stopped the weird messages.
I might need to check into this in the future.

File: packages/frontend/src/app/page.tsx [hunk 3]
```diff
@@ -16,24 +19,24 @@ import ChatsList from "../components/ChatsList";
...
+
+    getData();
+  }, []);
+
+  return (
+    <div className="flex w-screen h-screen">
+      <ChatsList onChatSelect={setSelectedChat} />
+      <MessagingPanel selectedFriend={selectedChat} />
+    </div>
+  );
```
Comment (mikolertesx) [inline packages/frontend/src/app/page.tsx:41]: This was auto-formatted, I hope this is the last time we deal with this; We can update the settings if you prefer to have them wider.

**Review by Lorenzo-Mariano — APPROVED** [review]
LGTM

**Pull request merged**